Activities/Database/UiPath.Database.Activities/UiPath.Database.Activities.csproj

```diff
@@ -71,6 +71,6 @@
     <Folder Include="Resources\" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Condition="'$(TargetFramework)' != 'net5.0'" Remove="NetCore/**" />
+    <Compile Condition="'$(TargetFramework)' == 'net461'" Remove="NetCore/**" />
```

Comment:
Lazy comment without checking how it currently appears in Visual Studio: in case this line removes the .net core files from the Solution Explorer in Visual Studio, add 
`<None Include="NetCore\**\*.cs" />`